### PR TITLE
Pod early launch fixes

### LINKED ISSUE #14294: escape pod problem
Step 1: be on an escape pod traveling to centcom

Step 2: send pod to mining while in transit to centcom

Step 3: escape pod gets sent to mining but your mob stays on the travel screen in space so you die

Maybe disable being able to send the pod to mining after it has already left to centcom?

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Fixes pods launching from the asteroid at round end

Fixes launching pods to the asteroid mid transit

Fixes #14294

### CODE REVIEW

**Comment by tkdrg** [comment]
Couldn't you add a var `early_launched` or something instead of relying on z level logic?

**Comment by Aranclanos** [comment]
these pod code is a mess

**Comment by KorPhaeron** [comment]
@tkdrg 

An early_launched var wouldn't help with the issue of launching to the asteroid mid escape transit

**Comment by tkdrg** [comment]
Then a `launched` var?

**Comment by KorPhaeron** [comment]
@tkdrg 

It'd need at least two vars, launched (to see if it was in transit, and not able to be sent to the asteroid), and early launched (to see if it had gone to the asteroid, and not eligible to be moved round end).

**Comment by tkdrg** [comment]
Makes sense I guess

**Comment by KorPhaeron** [comment]
Actually I guess defines for UNLAUNCHED LAUNCHED and EARLY_LAUNCHED would work for one var

**Comment by KorPhaeron** [comment]
@tkdrg

File: code/controllers/subsystem/shuttles/emergency.dm [hunk 1]
```diff
@@ -116,7 +120,8 @@
 			if(time_left <= 0 && !SSshuttle.emergencyNoEscape)
 				//move each escape pod to its corresponding transit dock
 				for(var/obj/docking_port/mobile/pod/M in SSshuttle.mobile)
-					if(M.z == ZLEVEL_STATION) //Will not launch from the mine/planet(for some reason)
+					if(!M.launch_status) //Will not launch from the mine/planet
```
Comment (tkdrg) [inline code/controllers/subsystem/shuttles/emergency.dm]: Could you make this one == UNLAUNCHED instead,  for clarity?

**Comment by KorPhaeron** [comment]
@tkdrg

**Pull request merged**